opm/parser/eclipse/Deck/Deck.hpp

```diff
@@ -187,6 +163,9 @@ namespace Opm {
             std::string input_path;
             DeckTree file_tree;
             mutable std::size_t unit_system_access_count = 0;
+
+            const DeckView& global_view() const;
+            mutable std::unique_ptr<DeckView> global_view__{nullptr};
```

Comment:
Note: I did say two consecutive underscores **anywhere**, not just two **leading** underscores.  In other words, user code isn't really supposed to use identifiers that

1. `__look_like_this`
2. `or_like__this`
3. `or_like_this__`

I'm guilty of this mistake myself, e.g., in the original version of the [`OPM_THROW`](https://github.com/OPM/opm-common/blob/90e55025d6246287c3e63ceb33e0457cc2f02a35/opm/common/ErrorMacros.hpp#L54) macro but I do think we should avoid adding more of these conformance issues.